### ISSUE TITLE
Identity Nullification

### ISSUE THREAD

**Issue by never2average** [Devsoc-BPGC/BetterHire#1]
The recruiter should not have access to your identity in terms of color/caste/religion/gender/orientation and so on in order to prevent any sort of intrinsic bias in the mind of the interviewer.
In order to do so, we have to convert the CV into a neutral format so that the machine that screens these CVs cannot make any decisions. Also, given a bunch of CVs, we need to be able to rank them.

**Comment by never2average** [comment]
Once we have discussed this issue, we will break it down into sub-issues. This is just the topic.

**Comment by yashagarwal1999** [comment]
Hey @never2average. If we mask or hide the applicant's name from the CV we can eliminate color/caste/religion/gender/orientation bias.

**Comment by never2average** [comment]
Yes, but from other texts as well, we need to remove pronouns too

**Comment by never2average** [comment]
The candidate picture needs to be removed as well. If present.

**Comment by AmoghSinha** [comment]
Suppose the recruiter contains some pre-conceived notions about a particular country. And some applicant is from that said country. Keeping this scenario in mind, is it possible to remove the column of college or country?
Thanks.

**Comment by sarveshwar22** [comment]
The candidate previous background also needs to be removed.

**Comment by never2average** [comment]
@AmoghSinha that is possible definitely, so the college can be replaced by a metric(ex: that represents its international rank) and the relocation cost the company for that candidate(the main reason why country might be important)

**Comment by never2average** [comment]
@sarveshwar22 what previous background are you trying to remove and how would this help seem unclear?

**Comment by TarangAgarwal** [comment]
> @AmoghSinha that is possible definitely, so the college can be replaced by a metric(ex: that represents its international rank) and the relocation cost the company for that candidate(the main reason why country might be important)

Does something similar also need to be done for the previous company(s) the applicant has worked for?

**Comment by never2average** [comment]
@TarangAgarwal Can be a possibility

**Comment by shubhamrgandhi** [comment]
Although professionals do not include photographs, we could double-check just to be sure and remove the photographs from the resume, if there and notify the applicant about the same. Just an idea.
